openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -513,6 +523,14 @@ def _get_user_and_profile(username):
     return existing_user, existing_user_profile
 
 
+def check_html(name):
```

Comment:
Two choices here:

Make it specific to the field `validate_name(name)`
Make it more generic `contains_html(value)`